rate-providers/MorphoERC4626RateProviders.md

```diff
@@ -68,6 +73,18 @@ If none of these is checked, then this might be a pretty great Rate Provider! If
     #### IndexCoop mhyETH (wETH) v1.1
     For [IndexCoop mhyETH](https://etherscan.io/address/0x701907283a57FF77E255C3f1aAD790466B8CE4ef) some allocators are eoas.
     - 0x622271CEC64F6106020F31773Ec5293F4677cb95
+    #### Seamless USDC v1.1
+    For [Seamless USDC](https://basescan.org/address/0x616a4e1db48e22028f6bbf20444cd3b8e3273738) some allocators are eoas.
+    - 0x8a438a7fb092E5D074a7aDe03E7eD25015817c58
+    #### Spark USDC v1.1
+    For [Spark USDC](https://basescan.org/address/0x7BfA7C4f149E7415b73bdeDfe609237e29CBF34A) no allocators are eoas. Only one allocator exists, which is a multi-sig.
+    #### Ionic USDC v1.1
+    For [Ionic USDC](https://basescan.org/address/0x23479229e52Ab6aaD312D0B03DF9F33B46753B5e#events) some allocators are eoas.
+    - 0x1f9A8c72327242b64bd21945E9750F5C5F50Eee1 
+    #### Ionic WETH v1.1
+    For [Ionic WETH](https://basescan.org/address/0x5A32099837D89E3a794a44fb131CBbAD41f87a8C) some allocators are eoas.
+    - 0x5f0761eed6Cd7F0Bd91d847E955cD78Bb50B0647
+
```

Comment:
I pattern matched this info by looking at `SetIsAllocator` events, but I'm not sure it is all that valuable since allocators can be added / removed by contract owner at any time, which means the info we are sharing here could be out of date

```
    function setIsAllocator(address newAllocator, bool newIsAllocator) external onlyOwner {
        if (isAllocator[newAllocator] == newIsAllocator) revert ErrorsLib.AlreadySet();

        isAllocator[newAllocator] = newIsAllocator;

        emit EventsLib.SetIsAllocator(newAllocator, newIsAllocator);
    }
```